frontend/public/components/alert-manager.tsx

```diff
@@ -83,37 +84,72 @@ export const AlertManagersListContainer = props => <Firehose resources={[{
   <AlertManagersNameList {...props} />
 </Firehose>;
 
-const AlertManagerRow = ({obj: alertManager}) => {
+const tableColumnClasses = [
+  classNames('pf-m-2-col-on-lg', 'pf-m-3-col-on-md', 'pf-m-6-col-on-sm'),
+  classNames('pf-m-2-col-on-lg', 'pf-m-3-col-on-md', 'pf-m-6-col-on-sm'),
+  classNames('pf-m-3-col-on-lg', 'pf-m-3-col-on-md', 'pf-m-hidden', 'pf-m-visible-on-md'),
+  classNames('pf-m-2-col-on-lg', 'pf-m-hidden', 'pf-m-visible-on-lg'),
+  classNames('pf-m-3-col-on-lg', 'pf-m-3-col-on-md', 'pf-m-hidden', 'pf-m-visible-on-md'),
```

Comment:
These classes don't match the column sizes from before. Was that intentional?

Reply:
same as above comment w/ bindings.tsx. Slight modifications due to slight column differences. I can revisit if needed.